doc/source/reliability/innodb_corrupt_table_action.rst

```diff
@@ -23,8 +23,7 @@ running in the background.
 
 If the :variable:`innodb_force_recovery` variable is set to a low value
 and you expect the server to crash,
-it may still be running due to
-a non-default value of the :variable:`innodb_corrupt_table_action` variable.
+the server may continue to run due to a non-default value of the :variable:`innodb_corrupt_table_action` variable.
```

Comment:
with innodb_force_recovery set to low value, the intention is to not crash. Infact it provides levels of recoverability after a crash and startup failures.

Please check this once: https://dev.mysql.com/doc/refman/8.0/en/forcing-innodb-recovery.html

Reply:
rewritten